scripts/pylib/twister/twisterlib/platform.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, data: dict[str, str]):
         self.exec = data.get("exec")
 
     def is_runnable(self) -> bool:
+        if self.exec == "simics":
+            return shutil.which("simics", path=os.environ.get("SIMICS_PROJECT")) is not None
```

Comment:
this only works for Linux like system.

Reply:
Why? Python [shutil doc](https://docs.python.org/3/library/shutil.html#shutil.which) is clear that it should work on Windows. And just in case, I tested if `shutil.which`, using a path from an env var, does work on Windows, and it does. 

Indeed, next line already uses `shutil.which`. What am I missing?